python/pytorchserver/pytorchserver/test_model.py

```diff
@@ -16,8 +16,9 @@
 import torch
 import torchvision
 import torchvision.transforms as transforms
+import os
 
-model_dir = "../../docs/samples/pytorch"
+model_dir = model_dir = os.path.join(os.path.dirname(__file__), "example_model")
```

Comment:
so somewhere the model got deleted in a refactor?

Reply:
No -- it's there, I just didn't want to couple docs to test.